doc/topics/windows/windows-package-manager.rst

```diff
@@ -890,23 +877,19 @@ definition file:
         uninstaller: '{B5B5868F-23BA-297A-917D-0DF345TF5764}'
         uninstall_flags: '/qn /norestart'
         msiexec: True
-        cache_file: salt://win/repo/MyApp/MyApp.1.1.msp
-
-There are a few things to note about this software definition file. First, is
-the solution we are trying to solve, that of applying a patch. Version ``1.0``
-just installs the application using the ``1.0`` MSI defined in the ``installer``
-parameter. There is nothing special in the ``install_flags`` and nothing is
-cached.
+        cache_file: salt://win/repo-ng/MyApp/MyApp.1.1.msp
 
-Version ``1.1`` uses the same installer, but uses the ``cache_file`` option to
-specify a single file to cache. In order for this to work the MSP file needs to
-be in the same directory as the MSI file on the ``file_roots``.
+In the above example:
+Version ``1.0`` of the software installs the application using the ``1.0`` MSI defined in the ``installer`` parameter.
+There is no file to be cached and the ``install_flags`` parameter does not include any special values.
 
-The final step to getting this to work is to add the additional ``/update`` flag
-to the ``install_flags`` parameter. Add the path to the MSP file using the
-``%cd%`` environment variable. ``%cd%`` resolves to the current working
-directory which is the location in the minion cache where the installer file is
-cached.
+Version ``1.1`` of the software uses the same installer file as Version ``1.0``.
+Now to apply patch to Version 1.0, make the following changes in the package definition file:
+- Place the patch file (MSP file) in the same directory as the installer file (MSI file) on the ``file_roots``
+- In the ``cache_file`` parameter, specify the path to single patch file.
+- In ``install_flags`` parameter, add the ``/update`` flag and include the path to the MSP file
+using the ``%cd%`` environment variable. ``%cd%`` resolves to the current working directory which is the location in the minion cache
```

Comment:
**Nitpick (grammar, non-blocking):** The preposition "which" always needs a comma in the word in front of it because "which" introduces non-restrictive clauses. https://www.scribbr.com/commas/comma-before-which/

Reply:
Updated